osbuild/mounts.py

```diff
@@ -123,8 +123,11 @@ def __init__(self, args):
         self.mountpoint = None
         self.check = False
 
+    # pylint: disable=[no-self-use]
     @abc.abstractmethod
     def translate_options(self, options: Dict) -> List:
+        if options.get("readonly", False):
+            return ["-o", "ro"]
         return []
```

Comment:
If a sub-class will try to append its own `-o [...]` options, we will have to check whether `mount(1)` merges the options or overrides them. However, I think we can defer this until it actually becomes an issue.

Reply:
Good point. And as you say, let's postpone this decision.

Reply:
Yeah, this is not the cleanest approach. I think we can fix in a follow up though.

Reply:
Interested to now what would make it cleaner though.